check_process

```diff
@@ -5,8 +5,8 @@
         admin="john"
         email="john@gmail.com"
         admin_email="john@gmail.com"
-        password="admin!42"
-        admin_password="admin!42"
```

Comment:
Are we using this variable (`admin_password`) somewhere?

Reply:
I didn't found any use of it. Well, let's push a commit that removes it.